app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+
+        self.distance_from_center = distance_from_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_of_cars: list[Car]) -> float:
+        income = 0.0
+        for car in list_of_cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        clean_possibility = (self.clean_power - car.clean_mark)
+        numerator = car.comfort_class * clean_possibility * self.average_rating
+
+        return round((numerator / self.distance_from_center), 1)
```

Comment:
The calculation of `clean_possibility` and `numerator` seems correct, but ensure that the division by `self.distance_from_center` does not lead to division by zero. Consider adding a check to prevent this scenario.